### PR TITLE
Weekly update for 2015-08-14

### PR DESCRIPTION
**Pull request by diagramatics**
In hopes of publishing this on time.

Key highlights:
- The move to nodejs/node is big news, so I put them in place of the usual updates of io.js versions, since there's none this week.
- Mentioned about nodejs/collaboration per #135.
- ~~Still waiting for Collaboration Summit's summary per nodejs/summit#13. Probably **do not merge yet** until a summary is available to post as a separate section or a decision is made to post a mention in the Community Updates section.~~ Solved per Mikeal's guidance, added a section on Community Updates with links to docs.

~~Once the summit concerns has been addressed~~, one LGTM is enough for me to immediately submit a PR to new.nodejs.org.

### CODE REVIEW

**Comment by yosuke-furukawa** [comment]
Thank you !!!!
I would like to add this post !!!! 
https://medium.com/node-js-javascript/4-0-is-the-new-1-0-386597a3436d

**Comment by diagramatics** [comment]
Done. :+1:

File: weekly-updates/weekly-update.2015-08-14.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* As mentioned up top, io.js v4.x will be Node.js v4.x. Or as [Mikeal Rogers](http://twitter.com/mikeal) said, [4.0 is the new 1.0](https://medium.com/node-js-javascript/4-0-is-the-new-1-0-386597a3436d).
+* The Node.js Collaborator Summit was hosted on 6th – 7th August in San Fransisco. You can see the notes in the [nodejs/summit](https://github.com/nodejs/summit) repository, [the GitHub issue](https://github.com/nodejs/summit/issues/11), or on [the Google Docs folder](https://drive.google.com/folderview?id=0B4tYrG9tjAW0fk9VOG52R2VQMmZHNWdpYnpqWU5MYUlfamNycktmeEl3UkNhdk1PS1Rid3c&usp=sharing).
+* [nodejs/collaboration](https://github.com/nodejs/collaboration) was made along with [the team](https://github.com/orgs/nodejs/teams/collaboration) to help with communication between Working Groups, as well as getting better ways to have evangelism, website, documentation and the 30+ i18n groups work together.
+* The new [Intl WG](https://github.com/nodejs/Intl) had [a meeting](https://github.com/nodejs/Intl/issues/8/?utm_source=io.js+and+Node.js+News&utm_medium=article) on August 11th.
```
Comment (yosuke-furukawa) [inline weekly-updates/weekly-update.2015-08-14.md]: > https://github.com/nodejs/Intl/issues/8/?utm_source=io.js+and+Node.js+News&utm_medium=article

The url has querystring for tracking source, we would be better to remove those queries.

**Comment by yosuke-furukawa** [comment]
LGTM, without tiny nits.
If you fix this, I will publish this PR to medium.

File: weekly-updates/weekly-update.2015-08-14.md (hunk 1 above)
Comment (diagramatics) [inline weekly-updates/weekly-update.2015-08-14.md]: It was regarding to #124, but maybe links to ourselves don't need them, you're right.

**Comment by diagramatics** [comment]
Alright, fixed. GitHub Desktop is surprisingly usable on mobile.

**Comment by diagramatics** [comment]
@yosuke-furukawa Seems like the Medium counterpart is missing a space, in "You can see the notes in thenodejs/summit repository" between the "the" and the "nodejs/summit" link.

**Comment by yosuke-furukawa** [comment]
thx

**Pull request merged**